FirebaseAppCheck/Tests/Unit/Core/FIRAppCheckIntegrationTests.m

```diff
@@ -62,6 +64,8 @@ @interface FIRAppCheckIntegrationTests : XCTestCase
 @property(nonatomic, nullable) id mockAppCheckProvider;
 @property(nonatomic, nullable) id mockTokenRefresher;
 
+- (void)testDefaultAppCheckProvider FIR_DEVICE_CHECK_PROVIDER_AVAILABILITY;
```

Comment:
@paulb777, I thought this was pretty clever. This test required DeviceCheck so rather than do a runtime check in the test method, do a compile time check. Now, you can tell whether DeviceCheck was available based on the test log.

Reply:
Maybe too clever for me. 😛 Might need to chat about this one.